src/DotVVM.Framework.Hosting.AspNetCore/ServiceCollectionExtensions.cs

```diff
@@ -27,10 +27,20 @@ public static class ServiceCollectionExtensions
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
         // ReSharper disable once InconsistentNaming
         public static IServiceCollection AddDotVVM<TServiceConfigurator>(this IServiceCollection services) where TServiceConfigurator : IDotvvmServiceConfigurator, new()
+        {
+            var configurator = new TServiceConfigurator();
+            return services.AddDotVvm(configurator);
+        }
+
+        /// <summary>
+        /// Adds DotVVM services with authorization and data protection to the specified <see cref="IServiceCollection" />.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="configurator">Configurator instance</param>
+        public static IServiceCollection AddDotVvm<TServiceConfigurator>(this IServiceCollection services, TServiceConfigurator configurator) where TServiceConfigurator : IDotvvmServiceConfigurator, new()
```

Comment:
The casing should be `AddDotVVM`.

Reply:
There is such a method, but it's private. I can make it public. `applicationRootPath`, `useErrorPages` and `modifyConfiguration` should be null as default

https://github.com/riganti/dotvvm/blob/78c2f51db091f991414f04748e2946f36f3b1b01/src/DotVVM.Framework.Hosting.Owin/AppBuilderExtensions.cs#L61